### DISCUSSION TITLE
sourcemaps showing even after productionBrowserSourceMaps: false,

### DISCUSSION THREAD

**Discussion by smalljoe22** [vercel/next.js#66411] [Help]
### Summary

Hello. I have a Next.js app and noticed that some of my code is visible in plain text in the source maps, which I did not want. Here is an image showing how it appears on my website:

![1](https://github.com/vercel/next.js/assets/138771260/8feaff86-6398-4ba5-b509-f87bb47c3347)

I have also seen on other websites that they only have one .js file and hide most of their data. How can I achieve the same?

Here is an example from another website:
![2](https://github.com/vercel/next.js/assets/138771260/bd3c05de-1ce7-46c2-8101-70c03e962c50)

Im using Next.js "^14.2.3"

Here is my next.config.js

```
/** @type {import('next').NextConfig} */

const nextConfig = {
  productionBrowserSourceMaps: false,
}

module.exports = nextConfig
```

**Comment by tosbourn** [comment]
@smalljoe22 did you get to the bottom of this?
